enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -29,7 +29,11 @@ status: implementable
 
 ## Summary
 
-We propose to extract metric storage, alerting and querying to a separate service that can be scaled, deployed and managed in a decoupled mode. This will allow robust multi-cluster, fleet monitoring with best experience possible for cluster admins. This means SD org, running managed HyperShift will use RHOBS managed by Observability Group and AppSRE. 
+This proposal aims to figure our monitoring case of HyperShift topology. There are three important parts of this proposal:
+
+For HyperShift Admin, we propose to extract metric storage, alerting and querying for Clusters managed by HyperShift to a separate service that can be scaled, deployed and managed in a decoupled mode. This will allow robust multi-cluster, fleet monitoring with best experience possible for cluster admins. This means SD org, running managed HyperShift will use the RHOBS managed by Observability Group and AppSRE. Metrics from control planes of hosted clusters will be shipped thanks to Monitoring Stacks.
+
+The data plane part of guest clusters requires changes too. In terms of feature parity for users custom or UWM monitoring and HPA we propose to deploy unchanged CMO for smallest changes possible. We also propose using CMO path to gather and send needed monitoring data for HyperShift Admin to RHOBs as well as data-plane telemetry through the usual path.
```

Comment:
Just so I'm clear, this is an unchanged CMO running on the management clusters, as opposed to the Prometheus Agent running on management clusters?  Nothing running on the guest workers, in either scenario.  Confirm? @ianbillett

Reply:
My understanding is that the proposal is to run CMO unchanged on the guest workers so that things like HPA can keep working and collectors like the node exporter and kube state metrics are laid down by the CMO.

Reply:
So two CMOs per "customer deployment"?

1. Standard CMO -> control plane
2. Standard CMO -> data plane

Reply:
I think @jan--f / @fpetkovski / @bwplotka will be able to help with this question?

Reply:
IIUC this is how HyperShift is deployed today:
* All monitoring components runs on the guest workers (unlike "traditional" installations of OCP where  the cluster monitoring operator runs on the control plane nodes).
* No service monitors, rules and dashboards are deployed for the control plane components (Kube/OpenShift API, scheduler, etcd, ...).

My understanding of the current proposal is the following:
* It introduces a new monitoring stack on the hosted control plane to collect metrics from the control plane components and forward them to RHOBS. The implementation is yet to be defined but it will eventually be based on the Prometheus agent.
* It doesn't change anything for the guest workers. The platform components running there are still monitored by CMO (required for HPA, console dashboards, telemetry, ...). A selected number of platform metrics could be forwarded to RHOBS if needed via remote write (already supported).
* It doesn't change anything for the management clusters which are standard OSD clusters.

Reply:
Amazingly explained @simonpasquier 

I updated the enhancement with some words from your explanation.